app/models/configured_system.rb

```diff
@@ -35,6 +37,9 @@ class ConfiguredSystem < ApplicationRecord
   delegate :my_zone, :provider, :zone, :to => :manager
   delegate :queue_name_for_ems_operations, :to => :manager, :allow_nil => true
 
+  virtual_delegate :cpu_total_cores, :to => :hardware, :allow_nil => true, :default => 0, :type => :integer
+  virtual_delegate :ram_size_mb,     :to => "hardware.memory_mb", :allow_nil => true, :default => 0, :type => :integer
```

Comment:
where are `cpu_total_cores, ram_size_mb` columns used ?  we are asking hardware to get those values in consumption without rollups.
`./app/models/chargeback/consumption_without_rollups.rb:58`

Reply:
The cpu_total_cores and ram_size_mb was added to expose the hardware info for Configured Systems in the UI but i backed these out and will create separate PR for that and UI.